html-forms.tar/html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email</p>
```

Comment:
this could be a `label` element so that it's associated with the email input on the next line.